smack-extensions/src/main/java/org/jivesoftware/smackx/mediaelement/element/MediaElement.java

```diff
@@ -87,6 +87,12 @@ public XmlStringBuilder toXML(XmlEnvironment xmlEnvironment) {
             .optAttribute("width", width)
             .rightAngleBracket();
 
+        if (!xmlEnvironment.getNamespace().contentEquals(NAMESPACE)) {
```

Comment:
Just to be sure that there are no NPEs, consider switching `getNamespace()` and `NAMESPACE`

Reply:
Makes sense ;)